pkg/names/names.go

```diff
@@ -31,3 +36,21 @@ const SERVICE_CA_CONFIGMAP = "openshift-service-ca"
 // MULTUS_VALIDATING_WEBHOOK is the name of the ValidatingWebhookConfiguration for multus-admission-controller
 // that is used in multus admission controller deployment
 const MULTUS_VALIDATING_WEBHOOK = "multus.openshift.io"
+
+// PROXY_TRUSTED_CA_CONFIGMAP is the name of the proxy.spec.trustedCA
+// ConfigMap that contains the trusted CA certificate bundle used for
+// proxying HTTPS connections.
+const PROXY_TRUSTED_CA_CONFIGMAP = "proxy-ca-bundle"
```

Comment:
we want to change this name, right?  trusted-ca-bundle or something?

Reply:
I'll update, I thought operators were already writing code based on the "proxy-ca-bundle" name.

Reply:
I updated the name to `trusted-ca-bundle` here and the proxy workflow gdoc.